### PR TITLE
Publish Keep Dashboard to GCS bucket on testnet

### PR DESCRIPTION
**Pull request by lukasz-zimnoch**
Here we change the Keep Dashboard testnet job to publish static files to the GCS bucket instead of building and pushing a Docker image. This change is a part of our shift from Kubernetes to GCS as a hosting platform for static websites/dapps.

### CODE REVIEW

File: .github/workflows/dashboard-testnet.yml
Comment (Shadowfiend) [inline .github/workflows/dashboard-testnet.yml:120]: We've generally been using a bucket name that matches the intended domain name here so the DNS setup can just CNAME to `c.storage.googleapis.com` (e.g. `preview.keep.network` as the bucket name, `preview.keep.network` CNAME `c.storage.googleapis.com` as the DNS record). I realize there's a possibility we may not strictly need this approach here since the testnet.keep.network DNS management is delegated to GCP, but it may still makes sense to stick to this pattern for consistency with the top-level domains and other deployments that use this strategy.

Will leave the ultimate decision up to you---just wanted to mention it before we finalized things here.

Comment (lukasz-zimnoch) [reply .github/workflows/dashboard-testnet.yml:120]: Yep I'm aware but if we want to use HTTPS we need to use a load balancer (https://cloud.google.com/storage/docs/hosting-static-website#lb-ssl). In that case, the bucket name doesn't matter as we set A records to the LB external IP.

Comment (lukasz-zimnoch) [reply .github/workflows/dashboard-testnet.yml]: Here it is https://github.com/keep-network/keep-core/pull/2562/commits/adcba5043b748a0299aa705eb593e045a85d2ee3

**Comment by nkuba** [comment]
LGTM, I'd like to check if it deploys correctly with ropsten deployment cycle before merging.

**Review by michalinacienciala — COMMENTED** [review]
I left one question. Other than that everything looks ok to me.

File: .github/workflows/dashboard-testnet.yml
Comment (nkuba) [reply .github/workflows/dashboard-testnet.yml:108]: Nice catch!

Comment (michalinacienciala) [reply .github/workflows/dashboard-testnet.yml:108]: In the future I will review our other workflows, as also some of them may have the redundant `npm ci` step...

Comment (lukasz-zimnoch) [reply .github/workflows/dashboard-testnet.yml]: Yep, you're right, it's redundant. Removed in https://github.com/keep-network/keep-core/pull/2562/commits/900cdceae314f10b6b554f19402519467cc58f3d

**Review by michalinacienciala — APPROVED** [review]

**Pull request merged**
